package.json

```diff
@@ -10,12 +10,23 @@
   },
   "packageManager": "pnpm@10.2.0",
   "scripts": {
-    "check:ci": "tsx cli/src/index.ts lint . && prettier . --check && syncpack lint && turbo run check types cf-typegen && git diff --exit-code && pnpm lint-npm-lockfiles",
+    "check": "turbo run build && pnpm run \"/^check:.*/\" && git diff --exit-code",
```

Comment:
When running turbo in CI, I'd recommend adding `--log-order=grouped` to make logs more readable.

Reply:
It says grouped is the default in CI. Are we not seeing that happen?

Reply:
Ah you know what, I never noticed the CI default there and have been passing it manually this whole time. Disregard!